eox_nelp/pearson_vue/pipeline.py

```diff
@@ -314,6 +315,13 @@ def get_exam_data(course_id, **kwargs):  # pylint: disable=unused-argument
     """
     courses_data = getattr(settings, "PEARSON_RTI_COURSES_DATA")
     exam_metadata = courses_data[course_id]
+
+    # This generates the clientAuthorizationID based on the user_id and course_id
+    exam_metadata["client_authorization_id"] = anonymous_id_for_user(
```

Comment:
```suggestion
    exam_metadata["@client_authorization_id"] = anonymous_id_for_user(
```
What do you think of adding the `@` char to know that is related and arg of tag? and not tag? of ead

Reply:
current implementation uses the `get_exam_data` result inside `import_exam_authorization ` pipe, probably this makes sense in that context, however if we consider the `get_exam_data`  method out of the context the `@` doesn't make sense and the result of this could be used in other pipes or methods which don't use the soap notation